lcx-api/README.md

```diff
@@ -0,0 +1,30 @@
+# Chainlink External Adapter for LCX api
+
+You must provide the api key using the env variable `API_KEY`!
+
+## Input Params
+
+- `symbol`: The pair to get the rates of (optional, default: all). Available choices:
+    - `all`
+    - `btc-usd`
+    - `btc-eur`
+    - `eth-usd`
+    - `eth-eur`
+
+## Output Format
```

Comment:
Could you please add an output example for `all` also. I think this should maybe be a different `endpoint` altogether, because `all` is not a specific symbol but a collection of all symbols, and the response is different.

Please take a look at the `coingecko` adapter for a multi endpoint adapter example.